docs/development/design/embed-api.rst

```diff
@@ -0,0 +1,211 @@
+Embed APIv3
+===========
+
+The Embed API allows to embed content from documentation pages in other sites.
+It has been treated as an *experimental* feature without public documentation or real applications,
+but recently it started to be used widely (mainly because we created a Sphinx extension).
+
+The main goal of this document is to design a new version of the Embed API to be more user friendly,
+make it more stable over time, support documentation pages not hosted at Read the Docs,
```

Comment:
Hey @agjohnson! 

> this language still sounds like "support for Sphinx"

I don't understand why it sounds like referring to Sphinx to you. Can you clarify this?

Also, I had some trouble understanding your suggestion "support embedding _outside_ documentation _hosted at_ Read the Docs". Mainly because it says "hosted at Read the Docs" _but_ with "outside" used as an adjective for "documentation". In any case, I found confusing how the sentence is written. I like the suggestion about using "embedding" but I'd like to keep it explicitly that it's "not hosted at Read the Docs" if that's possible.

I asked @astrojuanlu about this and he was also confused about how the sentence was written.

Is it possible to use something like the following? 

- "make it more stable over time, support embedding outside documentation not hosted at Read the Docs," (note that I re-added the _not_) or
- "make it more stable over time, support embedding documentation not hosted at Read the Docs," (note that I removed the _outside_ and added the _not_)

Reply:
I think Anthony is talking about using a client that isn't hoverxref to embed docs content in applications. I think the original copy was referring to having the Embed API embed content that we don't host.